components/button.css

```diff
@@ -0,0 +1,184 @@
+.btn {
+  background-color: transparent;
+  border: none;
+  cursor: pointer;
+  outline: none;
+  white-space: nowrap;
+}
+
+.btn:focus {
+  outline: 2px double var(--bg-primary);
+  outline-offset: 2px;
+}
+
+.btn i {
+  margin: 0 0.4rem;
+}
+
+.btn:not(.btn-icon) {
+  border: 1px solid transparent;
+  border-radius: 0.475rem;
+  padding: 0.7rem 0.8rem;
+  color: var(--white-text);
+  transition: background-color 0.2s linear;
+}
+
+.btn[disabled] {
+  opacity: 0.5;
+  pointer-events: none;
+}
+
+.btn-primary {
+  background-color: var(--bg-primary);
+}
+
+.btn-primary:hover {
+  background-color: #013f9b;
+}
+
+.btn.btn-primary-outline {
+  border: 1px solid var(--bg-primary);
+  color: var(--black-color);
+}
+
+.btn-primary-outline:hover {
+  background-color: var(--bg-primary);
+  color: var(--white-text);
+}
+
+.btn.btn-secondary {
+  background-color: var(--bg-secondary);
+  color: var(--black-color);
+}
+
+.btn-secondary:hover {
+  background-color: #6a8686;
+}
+
+.btn.btn-secondary-outline {
+  border: 1px solid var(--bg-secondary);
+  color: var(--black-color);
+}
+
+.btn.btn-secondary-outline:hover {
+  background-color: var(--bg-secondary);
+}
+
+.btn-success {
+  background-color: var(--btn-success-bgcolor);
+}
+
+.btn-success:hover {
+  background-color: var(--btn-success-hoverColor);
+}
+
+.btn-danger {
+  background-color: var(--btn-danger-bgcolor);
+}
+
+.btn-danger:hover {
+  background-color: var(--btn-danger-hoverColor);
+}
+
+.btn.btn-warning {
+  background-color: var(--btn-warning-bgColor);
+  color: var(--black-color);
+}
+
+.btn.btn-warning:hover {
+  background-color: var(--btn-warning-hoverColor);
+}
+
+.btn.btn-info {
+  background-color: var(--btn-info-bgColor);
+  color: var(--black-color);
+}
+
+.btn-info:hover {
+  background-color: var(--btn-info-hoverColor);
+}
+
+.btn.btn-l {
+  padding: 1.2rem 2rem !important;
+}
+
+.btn.btn-s {
+  padding: 0.5rem 0.6rem !important;
+}
+
+.btn.btn-link {
+  text-decoration: underline;
+  text-underline-offset: 2px;
+  text-decoration-color: var(--bg-primary);
+  color: initial;
+  font-size: 1.2rem;
+}
+
+.btn.btn-link:hover {
+  color: var(--bg-primary);
+}
+
+.btn.icon-btn-primary i {
+  color: var(--bg-primary);
+  font-size: 1.2rem;
+}
+
+/* toggle button */
+.toggle-btn {
+  box-sizing: border-box;
+  display: inline-block;
+  height: 40px;
+  width: 90px;
+  outline: 2px solid var(--black-color);
+  position: relative;
+}
+
+.toggle-btn input {
```

Comment:
This input related styling can be used inside other component but if it is rarely used you can keep it here else shift it to the other css file

Reply:
As of now not using this in many places so keeping here only.  Will shift this to other css file in future.